src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -105,6 +105,7 @@ export interface OrderMetaData {
   executedSellAmount: string
   executedSellAmountBeforeFees: string
   executedFeeAmount: string
+  executedSurplusFee: string
```

Comment:
`executedSurplusFee: string | null`

Sorry, maybe I forgot, but, why don't we use the interface from the SDK?

Reply:
P.s. added `totalFee` to SDK:
https://github.com/cowprotocol/cow-sdk/pull/88

Reply:
Nice! 

> executedSurplusFee: string | null

Why nullable?  As I see in the spec, it looks like is mandatory
https://api.cow.fi/docs/#/default/get_api_v1_orders__UID_

<img width="811" alt="image" src="https://user-images.githubusercontent.com/2352112/205685799-acd6ee65-643f-4874-8ba9-b862a6ecbafb.png">

Reply:
By fact it's nullable :) CHeck the real response from API for market orders

Reply:
Oops! Looks like an OpenAPI spec mistake. It is indeed nullable.

Reply:
> By fact it's nullable :) CHeck the real response from API for market orders

refuted by evidence :P